lib/virtual_regional_office/client.rb

```diff
@@ -11,6 +11,10 @@ def classify_contention_by_diagnostic_code(params)
       perform(:post, Settings.virtual_regional_office.ctn_classification_path, params.to_json.to_s, headers_hash)
     end
 
+    def get_max_rating_for_diagnostic_codes(params)
```

Comment:
For straightforward params, it's nice to have a method signature that's more self-descriptive (compare with the `generate_summary` method below). Maybe this method could take just the array, and build the hash inside.